src/platforms/android/include.gradle

```diff
@@ -0,0 +1,5 @@
+dependencies {
+	def androidxVersion =  project.hasProperty("androidxVersion") ? project.androidxVersion : "1.0.0"
```

Comment:
project property for androidx appcompat is called `androidXAppCompat` not `androidxVersion`